api/processors/auxil/log_event_creator.py

```diff
@@ -7,14 +9,20 @@
 )
 
 
+class FromOrToGroup(enum.Enum):
```

Comment:
Enums are kind of broken in Python, I would inherit from str as well, e.g. `class FromOrToGroup(str, enum.Enum)` or add a `__str__` magic method otherwise it might lead to bugs, for example, during serialization (though it is not the case right now).

Reply:
Yeah, I know about it, but now it's used only as a function param, so I decided that we don't need proper str representation for it.

Reply:
enum removed, so hiding this conversation